.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/pycqa/pylint
-    rev: v2.17.4
+    rev: v3.0.0
```

Comment:
use 3.1.0 or at least 3.0.3.

Could add the "too many arguments" disable in this file rather than in the source code.

Reply:
@dhalbert I thought the goal was to have all repos have the same flags. I'm happy to add it and remove the other changes. Let me know and will update

Reply:
ok, let's leave the pylint annotations that you added for now. Is pylint 3.1.0 OK?

Reply:
One new issue:
```
_bleio\common.py:239:12: R1737: Use 'yield from' directly instead of yielding each element one by one (use-yield-from)
```

Reply:
I can add an ignore or fix it (assuming it's supported)

Reply:
Updated and passing